packages/botonic-plugin-flow-builder/src/webview/use-webview-contents.ts

```diff
@@ -85,7 +84,7 @@ export function useWebviewContents<T extends MapContentsType>({
 
   useEffect(() => {
     const getResponseContents = async () => {
-      const url = `${apiUrl}/v1/bot_flows/${botId}/versions/${version}/webviews/${webviewId}/`
+      const url = `${apiUrl}/v1/bot_flows/${botId}/versions/${version}/webviews/${webviewId}/?provider_id=${providerId}`
```

Comment:
looking at how this looks, maybe it would make sense this parameter to be webchat_provider_id, as I think it's the onnly case where we will need to know something from this (in this case webchat preview) but let's leave it as it atm.